### DISCUSSION TITLE
Lots of input devices due to Weylus

### DISCUSSION THREAD

**Discussion by rinakra** [xournalpp/xournalpp#2978] [Q&A]
Every time I use the program Weylus  [https://github.com/H-M-H/Weylus](url) with Xournalpp it creates a _new_ input device, so the list of input devices in my Preferences is getting ridiculously long. 

How do I remove these devices so that they don't show up in my Preferences anymore?

**Comment by rolandlo** [comment]
These devices are listed in the `settings.xml` file. Under Linux (in version 1.1.0+dev) this file can be found in `~/.config/xournalpp/`. The tag starts with  
```
<data name="deviceClasses">
```
You can either remove the unwanted entries manually or delete the whole `settings.xml` file, so that a new one is created.

**Reply by rinakra** [reply]
Hm... I checked my .config files and I can't find one four xournalpp. Is it possible the file is soemwhere else? I am using an up-to-date Arch linux.

**Reply by rinakra** [reply]
Oh wait nvm. I found it, but it just wasn't in the .config folder. Thank you!
